com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJointContinuous.cs

```diff
@@ -67,11 +54,7 @@ public override float GetVelocity()
         /// <returns>floating point in Nm</returns>
         public override float GetEffort()
```

Comment:
Is there a point to this method, as ArticulationBody.jointForce will not return anything unless explicitly set by the user?

Reply:
Yeah really. I think this whole GetXXX thing needs to be redesigned, but that should be another card and another scope of work. There's quite a few that I'd change in the URDF importer.